BUG-Inputting wrong kwargs in CLI does not return warnings

I wanted to create a scenario-management template project using the CLI so I typed:
```bash
taipy create --teamplate scenario-management
```
Unfortunately, I wrote a typo, and wrote "teamplate" instead of "template" and I got the wrong template

Taipy's CLI should return warnings when inputting non-existing kwargs

Hello @AlexandreSajus, I am proposing some changes to the CLI. From PR #1052, the message will look something like this
```
$ taipy create --teamplate scenario-management
usage: taipy create [-h] [--template {scenario-management,default}]

options:
  -h, --help            show this help message and exit
  --template {scenario-management,default}
                        The Taipy template to create new application.
[2024-03-27 23:10:34.779][Taipy][ERROR] Unknown arguments: --teamplate. Did you mean: --template?
[2024-03-27 23:10:34.779][Taipy][ERROR] Please refer to the help message above for more information.
```

Basically, when a wrong argument is entered:
- first we print the help message of the command
- then notify the user of the wrong argument
- then if their is a typo that similar enough to an argument, there will be a suggestion

What do you think about it?

--- a/docs/relnotes.md
+++ b/docs/relnotes.md
@@ -46,6 +46,9 @@ This is the list of changes to Taipy releases as they were published.
 
 - In standalone job execution mode, the default value of *max_nb_of_workers* is now 2 instead of 1.
   For more information, please refer to [Job execution configuration](./manuals/core/config/job-config.md).
+- When using the Taipy command-line interface, if an unsupported argument is provided, the CLI
+  will display a message indicating the invalid argument. If the invalid argument is possibly a
+  typo, the CLI will suggest the closest valid argument.
 
 ## Significant bug fixes
 

